jenkins-integration/beaker/install/shared/install_deps.rb

```diff
@@ -0,0 +1,3 @@
+step "Install jq" do
+  install_package master, 'jq'
```

Comment:
The jq install appeared to work once I added the following step above the "Install jq" one:

```ruby
step "Configure epel" do
  tmp_module_dir = master.tmpdir('configure_epel')
  on(master, puppet('module', 'install', 'stahnma-epel', '--codedir', tmp_module_dir))
  on(master, puppet('apply', '-e', "'include epel'"))
end
```

This time, I saw the following in the console output when the job was run:

```
* Install jq

  puppetserver-perf-sut54.delivery.puppetlabs.net 08:53:39$ yum -y  install jq
    Loaded plugins: fastestmirror
    Loading mirror speeds from cached hostfile
     * base: mirrors.usc.edu
     * epel: mirrors.cat.pdx.edu
     * extras: mirrors.cat.pdx.edu
     * updates: repo1.sea.innoscale.net
    Resolving Dependencies
    --> Running transaction check
    ---> Package jq.x86_64 0:1.5-1.el7 will be installed
...
    Installed:
      jq.x86_64 0:1.5-1.el7

    Dependency Installed:
      oniguruma.x86_64 0:5.9.5-3.el7
    Complete!
```